src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Vector64.cs

```diff
@@ -313,6 +313,18 @@ public static unsafe Vector64<ulong> Create(ulong value)
         /// <returns>A new <see cref="Vector64{Byte}" /> with each element initialized to corresponding specified value.</returns>
         public static unsafe Vector64<byte> Create(byte e0, byte e1, byte e2, byte e3, byte e4, byte e5, byte e6, byte e7)
         {
+            if (Arm.AdvSimd.IsSupported)
+            {
+                Vector64<byte> result = Vector64.CreateScalarUnsafe(e0);
+                result = Arm.AdvSimd.Insert(result, 1, e1);
```

Comment:
I tried doing that, but I was getting build failures complaining about `AdvSimd` not found. It could be because my dev environment was not correct. I re-added the `using` and it is building fine now.